src/session/content/chat_action_bar.rs

```diff
@@ -217,15 +215,20 @@ impl ChatActionBar {
     }
 
     fn load_draft_message(&self, message: BoxedDraftMessage) {
-        // TODO: Load more message types
         let message_text = message
             .0
             .as_ref()
-            .map(|message| match message.input_message_text {
-                enums::InputMessageContent::InputMessageText(ref content) => {
-                    Cow::Borrowed(content.text.text.as_str())
+            .map(|message| {
+                if let InputMessageContent::InputMessageText(ref content) =
+                    message.input_message_text
+                {
+                    content.text.text.as_ref()
+                } else {
+                    unreachable!(
```

Comment:
Mh, maybe we should not do that, if Telegram suddenly decides to allow more draft messages Telegrand will just crash here.

Reply:
I think both are justified. The advantage of a crash is that the new functionality does not go unnoticed by us, as a crash would be reported immediately.

Reply:
Sure, but that's still not really great. Also, we can check for tdlib api changes in tdgrand anyway, so it would be easy to check for that.